readthedocs/templates/support.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load core_tags %}
+
+
+{% block title %}Support{% endblock %}
+
+{% block content %}
+<div id="support_page">
+{% filter restructuredtext %}
+
+Usage Questions
+~~~~~~~~~~~~~~~~
+
+If you have questions about how to use Read the Docs well,
+`Stack Overflow`_ is the best place to ask. Tag questions with ``read-the-docs`` so other folks can find them easily.
+
+{% if request.user.gold.count %}
+
+Priority Support
+~~~~~~~~~~~~~~~~
+
+Gold users have access to priority support for Read the Docs.
+You can contact us at {{ support_email }}.
```

Comment:
Did we settle on this? I'm not fond of this, I feel like companies will assume they can pay $20/mo for direct email support. I mean, again, not that they don't already assume this :)